chapter_convolutional-modern/vgg_vn.md

```diff
@@ -197,7 +198,9 @@ d2l.train_ch6(net, train_iter, test_iter, num_epochs, lr)
 * In their work Simonyan and Ziserman experimented with various architectures. In particular, they found that several layers of deep and narrow convolutions (i.e., $3 \times 3$) were more effective than fewer layers of wider convolutions.
 -->
 
-*dịch đoạn phía trên*
+* VGG-11 là một mạng sử dụng lại các khối tích chập. Các mô hình VGG khác nhau có thể được định nghĩa bằng sự khác biệt về số lượng các tầng chập và kênh đầu ra trên mỗi khối.
+* Việc sử dụng các khối tạo nên các biểu diễn rất nhỏ gọn trong việc định nghĩa mạng. Nó cho phép ta thiết kế hiệu quả các mạng phức tạp.
+* Simonyan và Ziserman đã thử nghiệm nhiều kiến trúc khác nhau. Cụ thể, họ đã phát hiện ra rằng một số tầng tích chập sâu và hẹp (ví dụ $3 \times 3$) có hiệu quả cao hơn so với các tầng tích chập rộng.
```

Comment:
Cụ thể, họ đã phát hiện ra rằng một số tầng tích chập sâu và hẹp (`$3 \times 3$`) có hiệu quả cao hơn so với các tầng tích chập rộng.

Reply:
`i.e.` means `that is`, nên nó hơi khác một chút so với `e.g.`. Mình thấy có thể bỏ dịch từ đó đi sẽ sát nghĩa hơn.

Reply:
```suggestion
* Simonyan và Zisserman đã thử nghiệm nhiều kiến trúc khác nhau. Cụ thể, họ đã phát hiện ra rằng nhiều tầng tích chập sâu và hẹp (như $3 \times 3$) thì có hiệu quả cao hơn so với một ít các tầng tích chập rộng.
```